packages/siopv2-oid4vp-op-auth/src/services/Siopv2MachineService.ts

```diff
@@ -215,7 +215,7 @@ export const translateCorrelationIdToName = async (correlationId: string, contex
     filter: [{ identities: { identifier: { correlationId } } }],
   })
   if (contacts.length === 0) {
-    return Promise.reject(Error(`Unable to find contact for correlationId ${correlationId}`))
+    return correlationId
```

Comment:
Whut?!?!

Reply:
Reverted. My guess is it was crashing the flow before the AddContact was working properly.
Will have to retest to see if this is bugging me.

Reply:
Yeah, means you should read these specs, because it should be clear